### PR TITLE
DHS-5:configuring maven artifacotry With Jfrog

### PR DESCRIPTION
**Pull request by sherrif10**
This is WIP  and it need to first test by uploading ref-distro properties in our jfrog repositoey
 cc @prapakaransp  @sharvanan

### CODE REVIEW

**Comment by sherrif10** [comment]
Hello Friends according to the discussion here , i have tried out this however am still trying to upload artifacts to jfrog to see if it functions as expected, Feel free to take a look https://digitalhealthcaresociety.jfrog.io/ui/repos/tree/General/digitalhealthsociety_repository

**Comment by sherrif10** [comment]
Thanks for working on this,hopefully will try to test it

**Comment by sherrif10** [comment]
cc @prapakaransp  @sharvanan  do you have some time to review this, thanks

**Review by prapakaransp — APPROVED** [review]
Good work Sharif. Can we merge and test it this.

File: pom.xml [hunk 1]
```diff
@@ -1213,16 +1225,16 @@
 		</pluginRepository>
 	</pluginRepositories>
 
-	<distributionManagement>
+<distributionManagement>
         <repository>
-            <id>openmrs-repo-releases</id>
-            <name>Releases</name>
-            <url>http://mavenrepo.openmrs.org/nexus/content/repositories/releases</url>
```
Comment (prapakaransp) [inline pom.xml:1220]: Is all module moved to digital healthcare society(jforg) Or Do we still need to pull modules from opnemrs.

File: pom.xml [hunk 2]
```diff
@@ -1213,16 +1225,16 @@
...
+            <id>digitalhealth-repo-releases</id>
+            <name>digitalHealthcare-Releases</name>
+            <url>https://digitalhealthcaresociety.jfrog.io/artifactory/digitalhealthsociety_repository/modules</url>
         </repository>
         <snapshotRepository>
-            <id>openmrs-repo-snapshots</id>
-            <name>OpenMRS Snapshots</name>
-            <url>http://mavenrepo.openmrs.org/nexus/content/repositories/snapshots</url>
```
Comment (prapakaransp) [inline pom.xml:1225]: I am guessing we could pull customized modules from digital health society and remaining from openmrs(jfrog). Please let me know your comments.

File: pom.xml (hunk 1 above)
Comment (sherrif10) [reply pom.xml:1220]: Thanks this have been moved to digital health care  jfrog, that way we will handle al the modules that we need either from openmrs or from customised DHS

File: pom.xml (hunk 2 above)
Comment (sherrif10) [reply pom.xml:1225]: You are right however if we need to depend on openmrs modules, we will manually need to use our our customised modules, push them to jfrog for it to keep up to date, but all these modules must first be up to date with openmrs code base which i do most of days checking on master

Comment (sherrif10) [reply pom.xml:1225]: Hope it makes sense

**Pull request merged**